python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -2047,3 +2047,29 @@ def convert_broadcast_to(node, **kwargs):
     )
 
     return [tensor_node, expand_node]
+
+@mx_op.register("UpSampling")
+def convert_upsample(node, **kwargs):
+    """Map MXNet's UpSampling operator attributes to onnx's Upsample operator
+    and return the created node.
+    """
+    name, input_nodes, attrs = get_inputs(node, kwargs)
+
+    sample_type = attrs.get('sample_type', 'nearest')
+    sample_type = 'linear' if sample_type == 'bilinear' else sample_type
+    scale = convert_string_to_list(attrs.get('scale'))
+    scaleh = float(scale[0])
+    scalew = float(scale[0])
+    if len(scale) > 1:
+        scalew = float(scale[1])
```

Comment:
quick question: if you check length of scale inputs here and accordingly pass to `UpSamplingNearestExp` ?

Reply:
Assuming I'm not misunderstanding the question, I don't believe so. The checks here are just to correctly pass in the scales to ONNX's upsampling operator.